src/main/java/christmas/data/Foods.java

```diff
@@ -0,0 +1,83 @@
+package christmas.data;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Foods {
+
+    // 판매하는 음식들을 담은 enum
+    public enum FoodItem {
+        양송이수프("양송이수프", "Appetizer", 6000),
+        타파스("타파스", "Appetizer", 5500),
+        시저샐러드("시저샐러드", "Appetizer", 8000),
+        티본스테이크("티본스테이크", "MainCourse", 55000),
+        바비큐립("바비큐립", "MainCourse", 54000),
+        해산물파스타("해산물파스타", "MainCourse", 35000) ,
+        크리스마스파스타("크리스마스파스타", "MainCourse", 25000),
+        초코케이크("초코케이크", "Dessert", 15000),
+        아이스크림("아이스크림", "Dessert", 5000),
+        제로콜라("제로콜라", "Beverage", 3000),
+        레드와인("레드와인", "Beverage", 60000),
+        샴페인("샴페인", "Beverage", 25000);
+
+        private final String name;
+        private final String type;
+        private final int price;
+
+        FoodItem(String name, String type, int price) {
+            this.name = name;
+            this.type = type;
+            this.price = price;
+        }
+        // 음식 이름 반환
+        public String getName() {
+            return name;
+        }
+        // 음식 종류 반환
+        public String getType() {
+            return type;
+        }
```

Comment:
해당 getter는 외부에서 메뉴가 ~~타입인지 확인하는 용도 이외에는 사용되지 않는 것 같은데요.
그렇다면 캡슐화를 더 잘 지킬 수 있도록, 그리고 메뉴가 메뉴의 책임을 온전히 다할 수 있도록 getter의 사용보다는 예상되는 타입을 파라미터로 전달받아 해당 타입과 동일한지 반환해 주는 메서드만으로도 충분할 것 같다고 생각합니다.

```suggestion
        public boolean isTypeOf(String expectedType) {
            return this.type.equals(expectedType);
        }
```